WordPress/Classes/Services/PostService+Likes.swift

```diff
@@ -71,6 +74,25 @@ extension PostService {
         return [LikeUser]()
     }
 
+    /**
+     Fetches a list of users from Core Data that liked the post with the given IDs after the given Date.
+     
+     @param postID  The ID of the post to fetch likes for.
+     @param siteID  The ID of the site that contains the post.
+     @param after   Filter results to likes after this Date.
+     */
+    func likeUsersFor(postID: NSNumber, siteID: NSNumber, after: Date) -> [LikeUser] {
+        let request = LikeUser.fetchRequest() as NSFetchRequest<LikeUser>
+        request.predicate = NSPredicate(format: "likedSiteID = %@ AND likedPostID = %@ AND dateLiked > %@", siteID, postID, after as CVarArg)
```

Comment:
`AND dateLiked > %@",`
I think this would need to be `AND dateLiked < %@",` to get likers who liked later the value of `after`.